gui/src/api/DashboardService.tsx

```diff
@@ -5,16 +5,16 @@ import {
 } from '../../types/projectsTypes';
 import { FormStoryPayload } from '../../types/storyTypes';
 import {
-  CommentReBuildPayload,
   CommentReBuildDesignStoryPayload,
+  CommentReBuildPayload,
   CreatePullRequestPayload,
 } from '../../types/pullRequestsTypes';
 import { CreateOrUpdateLLMAPIKeyPayload } from '../../types/modelsTypes';
 import {
   CreateDesignStoryPayload,
   EditDesignStoryPayload,
 } from '../../types/designStoryTypes';
-import { authPayload } from '../../types/authTypes';
+import { authPayload, UserData } from '../../types/authTypes';
```

Comment:
can you move it to camelCase instead of PascalCase

Reply:
@jedan2506 Please refer to style guide. https://ts.dev/style/#identifiers